src/pages/steps/StepsForm.tsx

```diff
@@ -178,43 +200,70 @@ const StepsForm = ({
           visible={!!toast.message}
           onClose={() => toast.clear()}
         />
-        <Steps
-          configurations={configurations}
-          onChange={handleChange}
-          fieldLoading={fieldLoading}
-          onChangeSuccess={handleChangeSuccess}
-          offerId={offerId}
-          mainLanguage={offer?.mainLanguage}
-          scope={scope}
-          form={form}
-        />
+        {needsLocationMigration ? (
+          <Stack>
+            <Alert variant={AlertVariants.DANGER} marginY={5}>
+              <Trans
+                i18nKey="create.migration.alert"
+                components={{
+                  boldText: <Text fontWeight="bold" />,
+                }}
+              />
+            </Alert>
+            <Steps
+              configurations={[locationStepConfiguration]}
+              fieldLoading={fieldLoading}
+              offerId={offerId}
+              onChange={handleChange}
+              mainLanguage={offer?.mainLanguage}
+              scope={scope}
+              form={form}
+            />
+          </Stack>
+        ) : (
+          <Steps
+            configurations={configurations}
```

Comment:
I wonder if you need to add a new `Steps` component inside the `needsLocationMigration` condtional.
Maybe you could write something like this? 

```suggestion
            configurations={needsLocationMigration ? [locationStepConfiguration] : configurations}
```

Reply:
This was originally to not have the onChange/onHandleSuccess but ended up needing them anyway, will simplify the code